[1m[31mERROR   [0m test_regression:test_regression.py:162 Test error message